files/zh-cn/web/accessibility/keyboard-navigable_javascript_widgets/index.md

```diff
@@ -1,105 +1,76 @@
 ---
 title: 键盘导航的 JavaScript 组件
```

Comment:
Let's align the translation:

![image](https://github.com/mdn/translated-content/assets/15844309/59f870b3-eaa3-4380-83a4-9c97dfeeff99)

```suggestion
title: 键盘导航的 JavaScript 微件
```